edx_rest_framework_extensions/__init__.py

```diff
@@ -1,3 +1,3 @@
 """ edx Django REST Framework extensions. """
 
-__version__ = '1.11.0'  # pragma: no cover
+__version__ = '2.0.0'  # pragma: no cover
```

Comment:
FYI: I'm not precious about version numbers.  Using 2.0 to represent a backward incompatible change.